treeopt/modules/metamodell.py

```diff
@@ -0,0 +1,36 @@
+import smt.surrogate_models as smt
+
+def RBF(xt, yt):
```

Comment:
`function_names` should be lower case with optional underscores, class names should be `CamelCase`. Check out the Black formatter, I think it should fix stuff like this automatically. You can either setup a git hook to format the code before each commit, or integrate it into your favorite python IDE

Reply:
Done :)